src/HonzaBotner.Discord.Services/EventHandlers/ServerBoosterHandler.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using DSharpPlus.EventArgs;
+using HonzaBotner.Discord.EventHandler;
+using HonzaBotner.Discord.Services.Options;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace HonzaBotner.Discord.Services.EventHandlers
+{
+    public class ServerBoosterHandler : IEventHandler<GuildUpdateEventArgs>
+    {
+        private readonly ILogger<ServerBoosterHandler> _logger;
+        private readonly CommonCommandOptions _commonOptions;
+
+        public ServerBoosterHandler(ILogger<ServerBoosterHandler> logger, IOptions<CommonCommandOptions> commonOptions)
+        {
+            _logger = logger;
+            _commonOptions = commonOptions.Value;
+        }
+
+        public async Task<EventHandlerResult> Handle(GuildUpdateEventArgs eventArgs)
+        {
+            if ((eventArgs.GuildBefore.PremiumSubscriptionCount ?? 0) <=
+                (eventArgs.GuildAfter.PremiumSubscriptionCount ?? 0))
+                return EventHandlerResult.Continue;
```

Comment:
Maybe add check to see if the subscriber count is lower after

Reply:
What is the difference in contrast to this?

Reply:
Oh my bad, I thought it was ==....